test/models/PostModel/test.js

```diff
@@ -0,0 +1,267 @@
+const {
+  PostModel, faker, expect, sinon, Post, Mongoose, dropDatabase,
+} = require('test/testHelper');
+const _ = require('lodash');
+const { PostFactory, wObjectFactory } = require('test/factories');
+
+describe('Post Model', async () => {
+  describe('On aggregate', async () => {
+    let post, author, permlink, title, body, postsCount;
+    beforeEach(async () => {
+      author = faker.name.firstName();
+      permlink = faker.random.string(20);
+      title = faker.address.city();
+      body = faker.lorem.sentence();
+      postsCount = faker.random.number(30);
+      for (let i = 0; i < postsCount; i++) {
+        if (i === 0) {
+          post = await PostFactory.Create({
+            author, permlink, title, body,
+          });
+        } else {
+          await PostFactory.Create({
+          });
+        }
+      }
+    });
+    describe('On group stage case', async () => {
+      let result;
+      beforeEach(async () => {
+        result = await PostModel.aggregate([{
+          $group: {
+            _id: '$_id',
+            author: { $first: '$author' },
+            permlink: { $first: '$permlink' },
+            title: { $first: '$title' },
+            body: { $first: '$body' },
+          },
+        }]);
+      });
+      it('Should return right count posts', async () => {
+        const { posts } = result;
+        expect(posts.length).to.be.eq(postsCount);
+      });
+      it('Should return records with id, author, perlink, title and body ', async () => {
+        const { posts: [posts] } = result;
+        expect(posts).to.have.all.keys(
+          '_id', 'author', 'permlink', 'title', 'body',
+        );
+      });
+    });
+
+    it('Should return from request: author, title and body', async () => {
```

Comment:
remove empty string

Reply:
wrap the last 2 it cases in describe('On project'), and describe('On errors'), if you started to wrap something in describe cases, do it everywhere in parent describe